Automattic/studio
# Studio: Fix site url after import

Related to https://github.com/Automattic/dotcom-forge/issues/8462

## Proposed Changes
After import, the URLs are pointing to the old site, and not to the URL used by studio.
This pr fixes that, by replacing the URLs in the database.


### Relation with sqlite PR
1. `wp search-replace` among other queries, it runs `LIKE REPLACE` queries as well ( I manage to make it do that after adding DROP table if exists in each sql file of a jetpack backup and try to import that ).
2. This sql statement, makes search-replace to fail, and although it reports that it did replacements, it actually doesn't.
3. You can easily test that, if you navigate locally to `~/Studio/< a site>` and run `wp search-replace`.

## Testing Instructions



1. Create a new site in studio.
2. Create and download a jetpack backup via activity log.
3. Locate your Studio site locally. Navigate to `wp-content/mu-plugins/sqlite-database-integration/`, and ensure that you apply the changes under [this pr](https://github.com/WordPress/sqlite-database-integration/pull/149) in your local plugin.
4. Start Studio, using  `STUDIO_IMPORT_EXPORT=true` flag.
5. Navigate to import/export tab, and import the backup you have previously downloaded.
6. After import, export the database of your site.
7. Open the `.sql` file, and search for `baseurl` and/or `siteurl`. Ensure it's the URL of your local site. 

NOTE: Some URLs are not getting updated, but this is normal, as stated under [this issue](https://github.com/wp-cli/wp-cli/issues/4871).
 


## Pre-merge Checklist
